APMrover2/Log.cpp

```diff
@@ -391,25 +431,39 @@ void Rover::Log_Write_Home_And_Origin()
 
 const LogStructure Rover::log_structure[] = {
     LOG_COMMON_STRUCTURES,
-    { LOG_PERFORMANCE_MSG, sizeof(log_Performance), 
-      "PM",  "QIHIhhhBH", "TimeUS,LTime,MLC,gDt,GDx,GDy,GDz,I2CErr,INSErr" },
-    { LOG_STARTUP_MSG, sizeof(log_Startup),         
-      "STRT", "QBH",        "TimeUS,SType,CTot" },
-    { LOG_CTUN_MSG, sizeof(log_Control_Tuning),     
-      "CTUN", "Qhcchf",     "TimeUS,Steer,Roll,Pitch,ThrOut,AccY" },
-    { LOG_NTUN_MSG, sizeof(log_Nav_Tuning),         
-      "NTUN", "QHfHHbf",    "TimeUS,Yaw,WpDist,TargBrg,NavBrg,Thr,XT" },
-    { LOG_SONAR_MSG, sizeof(log_Sonar),             
-      "SONR", "QfHHHbHCb",  "TimeUS,LatAcc,S1Dist,S2Dist,DCnt,TAng,TTim,Spd,Thr" },
-    { LOG_ARM_DISARM_MSG, sizeof(log_Arm_Disarm),
-      "ARM", "QBH", "TimeUS,ArmState,ArmChecks" },
-    { LOG_STEERING_MSG, sizeof(log_Steering),             
-      "STER", "Qff",   "TimeUS,Demanded,Achieved" },
+    {
+        LOG_PERFORMANCE_MSG, sizeof(log_Performance),
+        "PM",  "QIHIhhhBH", "TimeUS,LTime,MLC,gDt,GDx,GDy,GDz,I2CErr,INSErr"
+    },
+    {
+        LOG_STARTUP_MSG, sizeof(log_Startup),
+        "STRT", "QBH",        "TimeUS,SType,CTot"
+    },
+    {
+        LOG_CTUN_MSG, sizeof(log_Control_Tuning),
+        "CTUN", "Qhcchf",     "TimeUS,Steer,Roll,Pitch,ThrOut,AccY"
+    },
+    {
+        LOG_NTUN_MSG, sizeof(log_Nav_Tuning),
+        "NTUN", "QHfHHbf",    "TimeUS,Yaw,WpDist,TargBrg,NavBrg,Thr,XT"
+    },
+    {
+        LOG_SONAR_MSG, sizeof(log_Sonar),
+        "SONR", "QfHHHbHCb",  "TimeUS,LatAcc,S1Dist,S2Dist,DCnt,TAng,TTim,Spd,Thr"
+    },
+    {
+        LOG_ARM_DISARM_MSG, sizeof(log_Arm_Disarm),
+        "ARM", "QBH", "TimeUS,ArmState,ArmChecks"
+    },
+    {
+        LOG_STEERING_MSG, sizeof(log_Steering),
+        "STER", "Qff",   "TimeUS,Demanded,Achieved"
+    },
```

Comment:
I would say this also only takes more space, not improving anything. @lucasdemarchi, @magicrub?